### PR TITLE
Voltage init integration

### PR DESCRIPTION
**Pull request by etiennehomer**

### CODE REVIEW

**Review by FranckLecuyer — APPROVED** [review]

File: docker-compose/study/docker-compose.override.yml [hunk 1]
```diff
@@ -297,3 +297,21 @@ services:
     command: --server.port=80 --spring.config.additional-location=/config/
     sysctls:
       - net.ipv4.ip_unprivileged_port_start=0 # for docker < 20.03.0
+
+  voltage-init-server:
+    image: gridsuite/voltage-init-server:latest
+    ports:
+      - 5038:80
+    volumes:
+      - $PWD/../../k8s/resources/study/config/voltage-init-server-application.yml:/config/specific/application.yml:Z
```
Comment (dbraquart) [inline docker-compose/study/docker-compose.override.yml:306]: this new file voltage-init-server-application.yml should be created within this PR ?

File: docker-compose/study/docker-compose.override.yml [hunk 2]
```diff
@@ -297,3 +297,21 @@ services:
...
+      - $PWD/../../k8s/resources/common/config/common-application.yml:/config/common/application.yml:Z
+      - $PWD/../../k8s/resources/study/config/voltage-init-server-config.yml:/home/powsybl/.itools/config.yml:Z
```
Comment (dbraquart) [inline docker-compose/study/docker-compose.override.yml:308]: this new file voltage-init-server-config.yml should be created within this PR ?

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: a potentiel change for missing k8s/resources/study/config new files
tests: not done

File: docker-compose/study/docker-compose.override.yml (hunk 2 above)
Comment (etiennehomer) [reply docker-compose/study/docker-compose.override.yml:308]: Yes done

File: docker-compose/study/docker-compose.override.yml (hunk 1 above)
Comment (etiennehomer) [reply docker-compose/study/docker-compose.override.yml:306]: Yes done

**Review by dbraquart — APPROVED** [review]
code: ok
tests: not done

**Pull request merged**